pkg/operation/botanist/component/seedadmissioncontroller/seedadmissioncontroller.go

```diff
@@ -459,31 +459,319 @@ func GetValidatingWebhookConfig(caBundle []byte, webhookClientService *corev1.Se
 			SideEffects:             &sideEffect,
 			TimeoutSeconds:          pointer.Int32(10),
 		}, {
-			Name: "validation.extensions.seed.admission.core.gardener.cloud",
+			Name: "validation.extensions.backupbucket.admission.core.gardener.cloud",
```

Comment:
Why do we now need one entry per resource?

Reply:
In Client config, the `path` is different for every webhook.
```
ClientConfig: admissionregistrationv1.WebhookClientConfig{
				CABundle: caBundle,
				Service: &admissionregistrationv1.ServiceReference{
					Name:      webhookClientService.Name,
					Namespace: webhookClientService.Namespace,
					Path:      pointer.String(extensionresources.BackupBucketWebhookPath),
				},
			},
```

Reply:
Ah, I see, makes sense, thank you!